libs/perun/components/src/lib/password-reset/password-reset.component.html

```diff
@@ -1,20 +1,20 @@
-<h1 class="page-title">{{'PASSWORD_RESET.TITLE' | customTranslate | translate}}</h1>
+<h1 class="page-title">{{'SHARED_LIB.PERUN.COMPONENTS.PASSWORD_RESET.TITLE' | customTranslate | translate}}</h1>
 <div class="card mt-2 table-size-fix">
   <div class="card-body">
     <table [dataSource]="dataSource" class="w-100" mat-table matSort matSortActive="value"  matSortDirection="asc"
```

Comment:
for this table you should probably implement the overflow ass well and set the some min margin as you did in all other tables